src/olympia/pages/templates/pages/shield_study_13.html

```diff
@@ -0,0 +1,63 @@
+<h1 id="what-will-happen-next">What will happen next</h1>
+<p><strong>If you agree</strong> to participate:</p>
+<ol>
+<li>We will add additional features to Firefox that provide informational content about the websites you visit.</li>
+<li>Continue using Firefox as you normally would. You may not notice any changes immediately.</li>
+<li>After 28 days, the study will end on its own. Your Firefox will go back to normal.</li>
+<li>During the study, we will ask you to fill out a few quick <strong>optional surveys</strong> to tell us about your experience. The surveys should take less than 5 minutes to complete.</li>
+</ol>
+<h1 id="leaving-the-study">Leaving the study</h1>
+<p>The study will expire on its own in 28 days. You may leave the study early. To do so, follow these steps:</p>
+<ol>
+<li>Type <strong>about:addons</strong> into the location bar, and press enter.</li>
+<li>Find <strong>Web Health Study</strong> in the extensions list.</li>
+<li>Click the remove button.</li>
+<li>If you opt out of the study early we will ask you to fill out a survey. We’re interested in hearing about your experience, even if you didn’t participate in the entire study.</li>
+</ol>
+<p>Opting out of a study does not prevent you from participating in future studies.</p>
+<h1 id="your-privacy">Your privacy</h1>
+<h2 id="all-shield-studies">All Shield Studies</h2>
+<p>Every Shield Study collects data about important study events, such as install, uninstall, daily usage, and end of study. Studies also include an optional survey.</p>
+<p>This data is associated with <a href="https://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/index.html">Firefox Telemetry collection</a> and treated in accordance with our <a href="https://www.mozilla.org/privacy/">Privacy Policy</a>.</p>
+<h2 id="this-particular-study">This Particular Study</h2>
+<p>In addition to the data collected by all Shield Studies, here are the key things you should know about what is happening when you participate in this study:</p>
+<ul>
+<li>We will record the number of times you visit certain pre-defined set of websites (only top-level hostnames will be collected, e.g. www.amazon.com)</li>
+<li>We will record information about your interactions with notifications coming from this study</li>
+<li>The study will not be active, and no information will be recorded, while using Private Browsing Mode</li>
+<li>We will not collect any identifying information such as names, phone numbers, or email addresses</li>
+<li>This study is covered under IRB #827604 from the University of Pennsylvania</li>
+</ul>
+<p>You can learn more about the data collection for this study, including the pre-defined list of websites, <a href="https://github.com/raymak/shield-internet-policy-awareness/wiki/Data-Collection">here</a>.</p>
```

Comment:
`here` as a link text is very bad for accessibility reasons, the link text should describe the target. Can we change that to something more useful, e.g `in our Wiki` or something like that?